docs/resources/link_level_interface_policy.md

```diff
@@ -0,0 +1,144 @@
+
+The configuration snippet below creates a Link Level Interface Policy with only required attributes.
+
+```hcl
+
+resource "aci_link_level_interface_policy" "example" {
+  name = "test_name"
+}
+
+```
+The configuration snippet below shows all possible attributes of the Link Level Interface Policy.
+
+!> This example might not be valid configuration and is only used to show all possible attributes.
+
+```hcl
+
+resource "aci_link_level_interface_policy" "full_example" {
+  annotation          = "annotation"
+  auto_negotiation    = "off"
+  description         = "description_1"
+  port_delay          = "50"
+  emi_retrain         = "enable"
+  fec_mode            = "auto-fec"
+  link_debounce       = "2001"
+  name                = "test_name"
+  name_alias          = "name_alias_1"
+  owner_key           = "owner_key_1"
+  owner_tag           = "owner_tag_1"
+  physical_media_type = "sfp-10g-tx"
+  speed               = "100G"
+  annotations = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+  tags = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+}
+
+```
+
+All examples for the Link Level Interface Policy resource can be found in the [examples](https://github.com/CiscoDevNet/terraform-provider-aci/tree/master/examples/resources/aci_link_level_interface_policy) folder.
+
+## Schema ##
+
+### Required ###
+
+* `name` (name) - (string) The name of the Link Level Interface Policy object.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the Link Level Interface Policy object.
+
+### Optional ###
+
+* `annotation` (annotation) - (string) The annotation of the Link Level Interface Policy object. This attribute is supported in ACI versions: 3.2(1l) and later.
+  - Default: `"orchestrator:terraform"`
+* `auto_negotiation` (autoNeg) - (string) The auto-negotiation mode of the Link Level Interface Policy object. Auto-negotiation is an optional function of the IEEE 802.3u Fast Ethernet standard that enables devices to automatically exchange information over a link about speed and duplex abilities.
+  - Default: `"on"`
+  - Valid Values: `"off"`, `"on"`, `"on-enforce"`.
+* `description` (descr) - (string) The description of the Link Level Interface Policy object.
+* `port_delay` (dfeDelayMs) - (string) The port delay when coming up (milliseconds) of the Link Level Interface Policy object. The value must be in the range 0 - 10000. This attribute is supported in ACI versions: 4.2(5k) and later.
+  - Default: `"0"`
+* `emi_retrain` (emiRetrain) - (string) The electromagnetic interference (EMI) retrain state the Link Level Interface Policy object. This attribute is supported in ACI versions: 4.2(7q) to 4.2(7w), 5.2(4d) and later.
+  - Default: `"disable"`
+  - Valid Values: `"disable"`, `"enable"`.
+* `fec_mode` (fecMode) - (string) The forwarding error correction (FEC) mode of the Link Level Interface Policy object. This attribute is supported in ACI versions: 1.3(1g) and later.
```

Comment:
Can we change this to -> The Forward Error Correction (FEC) mode of the Link Level Interface Policy object.